expan/core/statistics.py

```diff
@@ -473,6 +477,8 @@ def normal_difference(mean1, std1, n1, mean2, std2, n2, percentiles=[2.5, 97.5],
             absolute values. In	this case, the interval is mean-ret_val[0] to
             mean+ret_val[1]. This is more useful in many situations because it
             corresponds with the sem() and std() functions.
+        multi_test_correction (boolean): True or False whether the correction for multiple testing is needed.
+        num_tests: number of tests or reported kpis used for multiple correction (default: 1, no correction is done)
```

Comment:
how about "num_tests (integer): number of tests or reported kpis used for multiple correction. This value is only used if multi_test_correction=True"?